satellite/gracefulexit/endpoint.go

```diff
@@ -444,8 +444,27 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 			if err != nil {
 				if metainfo.NodeAlreadyExitsError.Has(err) {
 					// this will get re-tried
-					endpoint.log.Debug("node already exists in pointer.", zap.Error(err))
+					endpoint.log.Warn("node already exists in pointer.", zap.Error(err))
 
+					// need to check if the process that sends the transfer messages has completed. if so, we need to send a new request
+					processMu.Lock()
+					if !morePiecesFlag {
```

Comment:
Why would flag be false if the transfer has failed and the piece is still in the pending queue?

Reply:
I think `morePiecesFlag` is a confusing name. It represents when all the records have been read from the transfer queue and the initial transfer requests have been sent.  I'll rename

Reply:
changed to always process the retry